src/api/routes/status.ts

```diff
@@ -0,0 +1,60 @@
+import { TypeBoxTypeProvider } from '@fastify/type-provider-typebox';
+import { Type } from '@sinclair/typebox';
+import { FastifyPluginCallback } from 'fastify';
+import { Server } from 'http';
+
+export const StatusRoutes: FastifyPluginCallback<
+  Record<never, never>,
+  Server,
+  TypeBoxTypeProvider
+> = (fastify, options, done) => {
+  // TODO: Add a job-queue etag cache handler.
+  fastify.get(
+    '/',
+    {
+      schema: {
+        summary: 'Service Status',
+        description: 'Displays the status of the Token Metadata Service and its current workload',
+        tags: ['Status'],
+        response: {
+          200: Type.Object({
+            status: Type.String(),
+            tokens: Type.Optional(Type.Record(Type.String(), Type.Integer())),
+            token_contracts: Type.Optional(Type.Record(Type.String(), Type.Integer())),
+            job_queue: Type.Optional(Type.Record(Type.String(), Type.Integer())),
```

Comment:
Might also be useful to include service version similar to the API and `/v2/info`